app/src/main/kotlin/com/wire/android/ui/common/ClickableText.kt

```diff
@@ -137,7 +138,11 @@ class LinkSpannableString(source: CharSequence) : SpannableString(source) {
     }
 
     override fun setSpan(what: Any?, start: Int, end: Int, flags: Int) {
-        super.setSpan(what, start, end, flags)
```

Comment:
suggestion(tests): any test we can add to confirm this works?

Reply:
Done